src/main/java/storm/sqs/SqsQueueSpout.java

```diff
@@ -68,6 +68,7 @@
 	public SqsQueueSpout(String queueUrl, boolean reliable) {
 		this.queueUrl = queueUrl;
 		this.reliable = reliable;
+		this.queue = new LinkedBlockingQueue<>();
```

Comment:
Unfortunately, I don't think this works; I don't think that `LinkedBlockingQueue` is Kryo-serializable by default. So it should be created by the worker in `prepare()`. Which unfortunately means it can't be `final`.